WowPacketParser/Parsing/Parsers/UpdateHandler.cs

```diff
@@ -2554,7 +2723,9 @@ private static MovementInfo ReadMovementUpdateBlock(Packet packet, WowGuid guid,
                 moveInfo = MovementHandler.ReadMovementInfo(packet, guid, index);
                 var moveFlags = moveInfo.Flags;
 
-                for (var i = 0; i < 9; ++i)
+                var speeds = ClientVersion.AddedInVersion(ClientVersionBuild.V3_0_3_9183) ? 9 : 8;
```

Comment:
seems like a strange thing to change between 3.0.2 and 3.0.3 tbh

Reply:
Not sure if this was intended for 3.0.2 vs 3.0.3 as more TBC vs WOTLK.